BSP/Inc/BSP_Contactor.h

```diff
@@ -4,6 +4,24 @@
 #define BSP_CONTACTOR_H
 
 #include "common.h"
+#include "stm32f4xx_gpio.h"
+#include "stm32f4xx_rcc.h"
+
+typedef uint8_t CONT_CHOICE;
+static const CONT_CHOICE ARRAY_CONTACTOR = (1 << 0);
+static const CONT_CHOICE HVHIGH_CONTACTOR = (1 << 1);
+static const CONT_CHOICE HVLOW_CONTACTOR = (1 << 2);
+static const CONT_CHOICE ALL_CONTACTORS = ARRAY_CONTACTOR | HVHIGH_CONTACTOR | HVLOW_CONTACTOR;
+
```

Comment:
Lol, ik this was already changed, but another way of writing this is `typedef enum {ARRAY_CONTACTOR = 1, HVHIGH_CONTACTOR = 2, HVLOW_CONTACTOR = 4, ALLCONTACTORS = 7}`. This is similar to what we do for the Fault bitmap and watchdog bitmap. I think this would be cleaner and easier to understand.

Reply:
Also, this might be a bit nitpicky, but CONT_CHOICE is a weird name, you could just leave it as CONTACTOR. Up to you tho, not a necessary change.

Reply:
I think we could go with either. I don't see a point to change it again so probably just gonna leave it